source/help/getting-started/managing-members.md

```diff
@@ -45,14 +45,33 @@ The System Admin is typically a member of the IT staff and has all the privilege
 - Ability to change any setting on the Mattermost server available in the System Console
 - Ability to promote and demote other users to and from the System Admin role
 - Ability to make a user account Inactive and to reactivate an Inactive account
- 
 
 #### Inactive 
 
-A System Administrator has the ability to make a user account Inactive by going to **System Console** > **Teams**, and selecting a team to manage. From their they can go to the **User** page for the team, and manage a user's role. 
+A System Administrator has the ability to make a user account Inactive by going to **System Console > Teams**, and selecting a team to manage. From there they can go to the **User** page for the team, and manage a user's role. 
 
 When "Inactive" is selected, the user will no longer be able to log into the system. 
```

Comment:
Is an inactive user also removed from teams and private channels? My PR assumes yes, but I could be wrong.